src/garage/torch/algos/sac.py

```diff
@@ -462,7 +462,10 @@ def _evaluate_policy(self, epoch):
 
         """
         eval_episodes = obtain_evaluation_episodes(
-            self.policy, self._eval_env, num_eps=self._num_evaluation_episodes)
+            self.policy,
+            self._eval_env,
+            num_eps=self._num_evaluation_episodes,
```

Comment:
This will end up changing the output of the benchmark. While this is fine, I think that the algorithm implementer/benchmarker should be able to pass whether they want deterministic evaluation sampling or not.

Reply:
I update the algorithm interface that now accepts `deterministic_eval_sampling`, a boolean argument.

Reply:
Hmm I don't think a parameter is the correct thing to do here. A stochastic algorithm should be evaluated with stochastic samples.

Are there prior works people need to reproduce which evaluate SAC using deterministic samples?